app/views/users/new.html.erb

```diff
@@ -0,0 +1,37 @@
+<div class="col-6 mx-auto">
+<h1 class="text-center">Sign up</h1>
+
+<% if @user.errors.any? %>  <!--もし、エラーが出たら表示する,validatesでの定義も忘れずに-->
+  <div id="error_explanation" class="alert alert-danger">
+    <ul>
+      <% @user.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+    </ul>
+  </div>
+<% end %>
+
+<%= form_for @user do |f| %>
+    <div class="form-group">
+      <%= f.label :name %>
+      <%= f.text_field :name, class:"form-control" %>      
+    </div>
+    <div class="form-group">
+      <%= f.label :email %>
+      <%= f.text_field :name, class:"form-control" %>     
```

Comment:
This one should be email not name. Please see code below
<%= f.email_field :email, class:"form-control" %>